### ISSUE TITLE
Error "'array.array' object has no attribute 'tostring'"

### ISSUE THREAD

**Issue by rlpavel** [sonicwu/alfred-rgb2hex#5]
No matter what I enter (space or comma separated, etc.), I get this error: 
<img width="580" alt="Capture 08-09 at 1239" src="https://user-images.githubusercontent.com/18669007/183708709-9184ef3e-ebb0-41e3-9854-c2dc86c1174d.png">

I don't know Python well enough to know what causes that, since there's no obvious references to 'array.array' or 'tostring'.

**Comment by sonicwu** [comment]
Can you provide the version of macos and python you are using?

**Comment by rlpavel** [comment]
@sonicwu macOS 12.4. I don't use Python, so it's the one that's preinstalled - Python 3.9.13. I also noticed that `python` command doesn't work, I had to specifically type `python3 --version`.

**Comment by sonicwu** [comment]
@rl-pavel I have the same environment as you, but can't reproduce the issue with the latest version (2.0.0) of the workflow.

Please try the following steps to fix or get more informations:
1. Confirm your have installed the latest version (2.0.0) of rgb2hex workflow
2. If still not working, open the `Alfred Preferences`, goto the `Workflow` tab, selected the `RGB to Hex` workflow, click the Debug button on the upper right corner to open the debug dialog.
3. Try the workflow again, put the debug log here if still not working.

**Comment by rlpavel** [comment]
@sonicwu seems like it's something with icon generation in the `png.py` script 
```
[11:07:01.194] RGB to Hex[Script Filter] Queuing argument '1 1 1'
[11:07:01.270] RGB to Hex[Script Filter] Script with argv '(null)' finished
[11:07:01.274] ERROR: RGB to Hex[Script Filter] Code 1: .
11:07:01 workflow.py:2093 DEBUG    ---------- RGB to Hex (2.0.0) ----------
11:07:01 workflow.py:2114 ERROR    'array.array' object has no attribute 'tostring'
Traceback (most recent call last):
  File "/Users/pavelorel/Dropbox/App/Alfred.alfredpreferences/workflows/user.workflow.EB98934E-E585-4BA8-8E1E-AEDA6CBC7E6B/workflow/workflow.py", line 2107, in run
    func(self)
  File "/Users/pavelorel/Dropbox/App/Alfred.alfredpreferences/workflows/user.workflow.EB98934E-E585-4BA8-8E1E-AEDA6CBC7E6B/rgb2hex.py", line 30, in main
    icon_file = generate_icon(r, g, b)
  File "/Users/pavelorel/Dropbox/App/Alfred.alfredpreferences/workflows/user.workflow.EB98934E-E585-4BA8-8E1E-AEDA6CBC7E6B/rgb2hex.py", line 44, in generate_icon
    w.write(f, p)
  File "/Users/pavelorel/Dropbox/App/Alfred.alfredpreferences/workflows/user.workflow.EB98934E-E585-4BA8-8E1E-AEDA6CBC7E6B/png/png.py", line 619, in write
    nrows = self.write_passes(outfile, rows)
  File "/Users/pavelorel/Dropbox/App/Alfred.alfredpreferences/workflows/user.workflow.EB98934E-E585-4BA8-8E1E-AEDA6CBC7E6B/png/png.py", line 788, in write_passes
    compressed = compressor.compress(tostring(data))
  File "/Users/pavelorel/Dropbox/App/Alfred.alfredpreferences/workflows/user.workflow.EB98934E-E585-4BA8-8E1E-AEDA6CBC7E6B/png/png.py", line 196, in tostring
    return row.tostring()
AttributeError: 'array.array' object has no attribute 'tostring'
11:07:01 workflow.py:2136 DEBUG    ---------- finished in 0.013s ----------
[11:07:01.275] RGB to Hex[Script Filter] <?xml version="1.0" encoding="utf-8"?>
<items><item valid="no"><title>Error in workflow 'RGB to Hex'</title><subtitle>'array.array' object has no attribute 'tostring'</subtitle><icon>/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertStopIcon.icns</icon></item></items>
```

**Comment by sonicwu** [comment]
@rl-pavel It should be the last time we upgrade the workflow to support Python3 without upgrading the dependent PyPNG module. 

Now I have released the version 2.0.1 to fix this issue. Please download it and confirm the issue is fixed on your environment.

**Comment by rlpavel** [comment]
@sonicwu yup, that fixed it, thank you!
